portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,10 @@
 // }
 
 
-/** Fetches data from the servers and adds them to the DOM.*/
-function getServerData(funFactTrigger) {
+/** Fetches data from the servers and adds them to the DOM.
+    funFactTrigger was used when two places in index.HTML called this method so this parameter
+    shows which place called it to distribute data properly*/
```

Comment:
I'll still approve, but ideally this comment would say what funFactTrigger does, so callers would know how to use this function.  Something like "funFactTrigger is a boolean specifying whether this method will XXX or YYY".

Reply:
Ah yes that's more descriptive and succinct